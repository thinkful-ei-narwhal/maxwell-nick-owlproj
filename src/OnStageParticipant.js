import React from 'react';

export default function OnStageParticipant(props) {

  return (
    <div className='onStage-participant' key={props.id}>
      <header>
        <h1>{props.name}</h1>
      </header>
      <img src={props.avatar} alt="testing" />
    </div>
  )
}
