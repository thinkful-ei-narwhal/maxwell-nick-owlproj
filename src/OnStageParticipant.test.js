import React from 'react';
import ReactDOM from 'react-dom';
import errorRenderer from 'react-test-renderer';
import store from './store';
import OnStageParticipant from './OnStageParticipant';

/* SMOKE TEST *//* SMOKE TEST *//* SMOKE TEST *//* SMOKE TEST *//* SMOKE TEST *//* SMOKE TEST *//* SMOKE TEST */
it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<OnStageParticipant key={store.participants[0].id} name={store.participants[0].name} avatar={store.participants[0].avatar} />, div);
  ReactDOM.unmountComponentAtNode(div);
});
/* SMOKE TEST *//* SMOKE TEST *//* SMOKE TEST *//* SMOKE TEST *//* SMOKE TEST *//* SMOKE TEST *//* SMOKE TEST */

it('renders the UI as expected', () => {
  const JSON = errorRenderer
    .create(<OnStageParticipant key={store.participants[0].id} name={store.participants[0].name} avatar={store.participants[0].avatar} />)
    .toJSON();
  expect(JSON).toMatchSnapshot();
});