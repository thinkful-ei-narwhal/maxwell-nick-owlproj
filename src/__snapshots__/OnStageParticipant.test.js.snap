// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="onStage-participant"
>
  <header>
    <h1>
      Koren Templeton
    </h1>
  </header>
  <img
    alt="testing"
    src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
  />
</div>
`;
